web-app/src/components/Leave/index.jsx

```diff
@@ -3,7 +3,7 @@ import { PropTypes as PT } from 'prop-types';
 import container from './container';
 import { Card, Button, Modal } from '../common';
 
-export const Leave = props => {
+const Leave = props => {
```

Comment:
Exporting the component here would allow us to test the component itself without the wrapping container. In your test file you'd just have to do a named import e.g. `import { Leave } from 'xyz`

Reply:
@fraser-uno Nope, because it's the default that's being retrieved, and `Leave` is just what you're assigning it to.

Reply:
Oh right okay, cool. We can do this with all components then, whole lot easier

Reply:
@fraser-uno - you're right, sorry, should have been clearer. When you're importing it in the test file, import it from the `Leave` directory rather than the `components` one.

So `import { Leave } from 'components/Leave'`. That'll import the export named `Leave` from the `index.js` file in `components/Leave`

Just for reference, 
`import Leave from 'components/Leave'` would grab the default export, which in this case would be the wrapped component.

Our big file of exports is just so that we can import multiple components in one line. I'm not very good at explaining things so if you want me to run through it with you give me a shout.

Reply:
@bgribz no no that's a good explanation. First time I've come across project with default and named exports so it was bit confusing to get my head around!